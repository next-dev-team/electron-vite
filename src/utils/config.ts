type IConfig = {
  /** 基础路由配置，/结尾 */
  baseRouterPrefix: string;
  /** 接口api前缀 */
  baseApiPrefix: string;
};

const config: IConfig = {
  baseRouterPrefix: import.meta.env.VITE_BASE_ROUTER_PREFIX,
  baseApiPrefix: '/api/',
};

export default config;
