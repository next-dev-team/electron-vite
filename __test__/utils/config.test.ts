import { describe, expect, it } from 'vitest';
import config from '@/utils/config';

describe('@/utils/config.ts', () => {
  it('配置文件', () => {
    expect(config).toMatchSnapshot();
  });
});
