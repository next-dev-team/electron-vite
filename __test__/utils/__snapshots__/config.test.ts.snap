// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`@/utils/config.ts > 配置文件 1`] = `
{
  "baseApiPrefix": "/api/",
  "baseRouterPrefix": "/sub/",
}
`;
